ginepro/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ginepro"
-version = "0.3.1-alpha.0"
+version = "0.4.0-alpha.0"
```

Comment:
We can maybe just prepare for the `0.4.0` release here

Reply:
Sounds good